ibis/backends/pyspark/__init__.py

```diff
@@ -929,3 +927,186 @@ def to_pyarrow_batches(
         return pa.ipc.RecordBatchReader.from_batches(
             pa_table.schema, pa_table.to_batches(max_chunksize=chunk_size)
         )
+
+    @util.experimental
+    def read_csv_directory(
```

Comment:
I thought we had agreed on `read_*_dir`.

Reply:
Oh in the notes it says we agreed on having separate methods, for example, "`to_csv_dir`/`to_parqet_dir` (or much better named)" so I named them `to_*_directory`. But I'm happy to change it to `to_*_dir` if that's better, I don't feel strongly about this.

Reply:
Let's go with `dir` for now.